src/software/sensor_fusion/refbox_data.h

```diff
@@ -98,91 +97,156 @@ class TeamInfo
     TeamInfo(std::string name, int score, int red_cards,
              std::vector<int> yellow_card_times, int yellow_cards, int timeouts,
              int timeout_time, int goalkeeper, int foul_counter,
-             int ball_placement_failures, bool can_place_ball, int max_allowed_bots);
+             int ball_placement_failures, bool can_place_ball, int max_allowed_bots)
+        : name(name),
+          score(score),
+          red_cards(red_cards),
+          yellow_card_times(yellow_card_times),
+          yellow_cards(yellow_cards),
+          timeouts(timeouts),
+          timeout_time(timeout_time),
+          goalkeeper(goalkeeper),
+          foul_counter(foul_counter),
+          ball_placement_failures(ball_placement_failures),
+          can_place_ball(can_place_ball),
+          max_allowed_bots(max_allowed_bots)
```

Comment:
@MathewMacDougall @akhilveeraghanta - what are your thoughts on moving this to be header-only? I totally see why we might want to here, it's very small, but it's also a bit inconsistent with the rest of the codebase, and I feel that consistency is super important, especially for our team where even slight inconsistencies can really confuse new members.

Reply:
Yea I'd vouch for consistency here (so to **not** have it header-only)

Reply:
refbox_data.h is also header only. My thought is that these classes are basically constructors, getters, and comparators, so they don't "deserve" a source file and in theory could be autogenerated.

Reply:
Done